internal/config/defaults.go

```diff
@@ -23,24 +19,38 @@ var (
 
 	// operations holds the supported operations inside mesh
 	operations = map[string]interface{}{
-		InstallKuma: map[string]interface{}{
+		InstallKumav071: map[string]interface{}{
```

Comment:
Is a rename from `defaults.go` to `supported_operations` a straight-forward change? If so, doing so will enhance clarity of the purpose of this file.

Reply:
I agree. It would be nice if this could be uniform across adapters, in consul-adapter it is called `supported_ops`. `supported_operations` sounds fine :-)

Reply:
@leecalcote @mgfeller The issue with renaming is that everything inside config package is treated as an independent entity. Now if we name the file supported_operations, that means its tied towards the adaptor logic and no other config data will go into that file.

Reply:
The framework is rigid about the filename, huh? Operations aren't really config and are not defaults. We could import from another file, if the framework is that rigid.

Reply:
Let's get 1) SMI conformance completed  and 2) the OSM and learn-layer5 and tanzu-sm port thing figured out and released.

Reply:
I didn't see that it contains all default values, not just operations, and thus didn't understand the intention of the file, my bad. I agree with @leecalcote's comment, and don't quite understand why operations belong there.